server_test.go

```diff
@@ -58,3 +59,79 @@ func TestStateNameToInt(t *testing.T) {
 		assert.Equal(t, int64(0), stateInt)
 	})
 }
+
```

Comment:
You should move the new tests to `bucket_metrics_test.go` since you moved logic for it